docker-compose.yaml

```diff
@@ -1,51 +1,30 @@
-# Comments are provided throughout this file to help you get started.
-# If you need more help, visit the Docker compose reference guide at
-# https://docs.docker.com/compose/compose-file/
+version: '3.9'
 
-# Here the instructions define your application as a service called "server".
-# This service is built from the Dockerfile in the current directory.
-# You can add other services your application may depend on here, such as a
-# database or a cache. For examples, see the Awesome Compose repository:
-# https://github.com/docker/awesome-compose
 services:
-  server:
-    build:
-      context: .
-      target: final
+  tiny_site_backend:
+    container_name: tiny_site_backend
+    image: realdevsquad/tiny_site_backend:1.0.0
```

Comment:
Why are we getting the image from dokcer hub?
We should be building the image on user system it seld